.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: UI Tests
+
+on:
+    workflow_dispatch:
+    deployment_status:
+
+env:
+  GIT_PAT: ${{ secrets.GIT_PAT }}
+  MNEMONIC: ${{ secrets.MNEMONIC }}
+  MNEMONIC_MAINNET: ${{ secrets.MNEMONIC_MAINNET }}
+  MNEMONIC_METAMASK: ${{ secrets.MNEMONIC_METAMASK }}
+  PATH_COSMOSTATION: './wallets/cosmostation'
+  PATH_KEPLR: './wallets/keplr'
+  PATH_LEAP: './wallets/leap'
+  PATH_METAMASK: './wallets/metamask' 
+  PRIVATE_KEY_METAMASK: ${{ secrets.PRIVATE_KEY_METAMASK }}
+  PUBLIC_ADDRESS_CONTRACT_ETH: ${{ secrets.PUBLIC_ADDRESS_CONTRACT_ETH }}
+  WALLET_PASSWORD: ${{ secrets.WALLET_PASSWORD }}
```

Comment:
And these

Reply:
Think we can also remove Metamask related envs as we don't use Metamask in Duality

Reply:
wallet password is required. Why isn't it required?

Reply:
We don't use it after refactoring I did a month ago